lib/util/timer.js

```diff
@@ -101,3 +110,11 @@ shaka.util.Timer = class {
     }
   }
 };
+
+/**
+ * Called whenever any timer is ticked.
+ * This variable is meant for use in testing.
+ * @type {function()}
+ * @export
+ */
+shaka.util.Timer.onTimerTick;
```

Comment:
I don't especially like this being part of the production build.  Could we instead have something conditional on DEBUG being defined, and test against an uncompiled build?

Reply:
Done.